### PR TITLE
change to @ import

### PR DESCRIPTION
**Pull request by brian-ngyn**

### CODE REVIEW

File: app/src/renderer/code/App.tsx [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
+import TabsContainer from "@/components/containers/TabsContainer";
+import { useAppState } from "@/contexts/AppStateContext";
 
 function App() {
   const { currentAppState } = useAppState();
   // 38, 55, 7
   return (
     <div className={currentAppState.darkMode ? "dark" : ""}>
-      <div className="h-screen w-screen overflow-x-hidden bg-light p-4 text-light dark:bg-dark dark:text-dark">
+      <div className="size-screen overflow-x-hidden bg-light p-4 text-light dark:bg-dark dark:text-dark">
```
Comment (ideen1) [inline app/src/renderer/code/App.tsx:16]: Please remove general App staying changes. These will be handled in a different ticket

File: app/src/renderer/code/App.tsx [hunk 2]
```diff
@@ -26,8 +25,8 @@ function App() {
               </BrowserRouter>
             </div>
           </div>
-          <div className="flex flex-row flex-wrap gap-x-2 gap-y-2">
-            <div className="h-96 w-96 grow">
+          <div className="flex flex-row flex-wrap gap-2">
+            <div className="size-96 grow">
```
Comment (ideen1) [inline app/src/renderer/code/App.tsx:29]: Please remove general App staying changes. These will eb handled in a different ticket

File: app/src/renderer/code/components/atoms/FaultCard.tsx [hunk 3]
```diff
@@ -21,16 +19,16 @@ function FaultCard(props: FaultCardProps) {
       <div className="flex p-2">
         <div className="flex justify-self-start"></div>
         <div
-          className={`flex h-full w-full items-center justify-start border p-2 ${
-            severity == ISeverity.ERROR
+          className={`flex size-full items-center justify-start border p-2${
```
Comment (ideen1) [inline app/src/renderer/code/components/atoms/FaultCard.tsx:22]: Please remove general App staying changes. These will eb handled in a different ticket

**Review by ideen1 — CHANGES_REQUESTED** [review]
Please ensure the styling of the site is identical to what is currently on main.

**Review by ideen1 — CHANGES_REQUESTED** [review]
Looks good. Thanks for making the styling changes. As per @CalebBourb, please add the end of file new lines to our TS config. As discussed today, this may already be implemented, so instead can you add a husky hook to run prettier --write .? Let me know if you need any support with this. Once this is in, we can all rebase off of this awesome work and use all of the styling enforcements :)

**Comment by ideen1** [comment]
@UCSolarCarTeam/telemetry Please review this PR ASAP so we can get all our prettier/ ts rules in :) 

Feel free to point out anything you notice in terms of code styling, logic or questions you have about the code!!

**Comment by brian-ngyn** [comment]
@UCSolarCarTeam/telemetry cherry pick 83e0f3b999c7e3aa4fc2c6891214c7c1202c637d into your current branches. You should run the following command from the Helios-Telemetry folder

```
git cherry-pick 83e0f3b999c7e3aa4fc2c6891214c7c1202c637d
```

Make sure to update your dependencies by running `npm install`
Also make sure you have Prettier and ESLint extensions installed

**Review by ideen1 — APPROVED** [review]
All looks good to me

**Review by justin-phxm — APPROVED** [review]
ok

**Review by CalebBourb — APPROVED** [review]
WOW i am amazed by the quality of this code! I wishto model my life after your work, it truly is an inspiration. 5/5

**Pull request merged**